### DISCUSSION TITLE
Portal access for self-hosting

### DISCUSSION THREAD

**Discussion by adimyth** [svix/svix-webhooks#575] [Q&A] [answered]
Thanks to the Svix team for this wonderful product. I have been playing with it this last week. I have deployed the containerised service & am trying to get access to portal.

The `Get App Portal Access` api returns a URL which points to [app.svix.com](app.svix.com), which of course isn't loading up & throws an `authentication failed message`. How can I access the portal?

If portal isn't shipped as part of docker container, then can we mention it explicitly somewhere & remove it from the api schema?

**Comment by tasn** [comment, marked as answer]
Thanks @adimyth, can you please open a ticket about this? I guess you're right and it should be removed from the schema for the self-hosted service (well, we'll just put a placeholder URL). We just didn't want to break the schema, and getting the token is still very valuable because you can build your own UI.
